tutorials/hist/Normalizing_Histogram.C

```diff
@@ -0,0 +1,171 @@
+/// \file
+/// \ingroup tutorial_hist
+/// \notebook -js
+/// Normalizing a Histogram
+///
+/// Image produced by `.x Normalizing_Histogram.C`
+/// 7 different methods of normalizing Histograms
+/// are shown, each with the original Histogram
+/// next to the normalized one.
+/// \macro_image
+/// \macro_code
+///
+/// The methods are:
+/// 1:
+/// ```
+/// Double_t num = h->GetBinContent(i);
+/// Double_t den = h->GetBinWidth(i);
+/// Double_t value = 0;
+/// if (den!=0)
+/// {
+///  value = num/den;
+///  h->SetBinContent(i,value);
+/// }
+///```
+/// 2:
+/// ```
+///  Double_t factor = 1.;
+///  h->Scale(factor/h->GetEntries());
+/// ```
+/// 3:
+/// ```
+/// Double_t scale = h->GetXaxis()->GetBinWidth(1)/(h->Integral());
+/// h->Scale(scale);
+/// ```
+/// 4:
+///```
+/// Double_t factor = 1.;
+/// h->Scale(factor/h->Integral());
+///```
+/// 5:
+///```
+/// Double_t factor = 1.;
+/// h->Scale(factor, "width");
+/// ```
+/// 6:
+/// ```
+/// Double_t factor = 1.;
+/// h->Scale(factor/h->Integral(), "width");
+/// ```
+/// 7:
+///```
+/// Double_t factor = 1.;
+/// h->Scale(factor/h->Integral("width"));
+///```
+///
+/// You can choose whatever method works best for you
+///
+/// \author Advait Dhingra
+
+#include "TH2F.h"
+#include "TRandom.h"
+#include "TCanvas.h"
+
+void Normalizing_Histogram()
+{
+
+   // all of the histograms
+
+   TH1F *hist1 = new TH1F("Unnormalized Histogram", "Unnormalized Hist", 100, 0, 100);
+   hist1->SetStats(0);
```

Comment:
@couet do we need this?

Reply:
I thought it makes the whole thing cleaner